lrlex/src/lib/lexer.rs

```diff
@@ -252,9 +256,16 @@ impl<'a, StorageT: Copy + Eq + Hash + PrimInt + Unsigned> Lexer<StorageT>
             )
         }
 
-        let (line_idx, col_byte) = line_col_off(self, i);
-        let line = self.surrounding_line_str(i);
-        (line_idx, line[..col_byte].chars().count() + 1)
+        fn line_col<StorageT: Copy + Eq + Hash + PrimInt + Unsigned>(
```

Comment:
Does this mean we are defining a function `line_col` within another function also called `line_col`? If so that's a little bit confusing.

Reply:
It's a local function so the two can't be confused. Maybe I could call the inner functions `lc`?

Reply:
Changed in https://github.com/softdevteam/grmtools/pull/152/commits/23b18e6324a0649930b7481dce1197d53633d733. See if you think it's better or not!

Reply:
I noticed it's in inner function but I still found it confusing that it had the same name. Renaming it is a good idea IMO.